<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createaccount</class>
 <widget class="QDialog" name="createaccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-190</y>
     <width>1251</width>
     <height>911</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/back.png</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
   </property>
   <property name="title">
    <string>Cr√©er un nouveau compte :</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>veuiller remplir tous les champs :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>cin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nom">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prenom">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_role">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>role</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_mdp">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_id">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_role">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>216</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>choisir</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>expert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>employee</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
